met-web/package.json

```diff
@@ -1,94 +1,133 @@
+    "dependencies": {
+        "@emotion/react": "^11.9.0",
+        "@emotion/styled": "^11.8.1",
+        "@formio/react": "^5.2.1",
+        "@mui/icons-material": "^5.8.3",
+        "@mui/material": "^5.6.1",
+        "@reduxjs/toolkit": "^1.8.1",
+        "@testing-library/react": "^13.3.0",
+        "@testing-library/user-event": "^13.5.0",
+        "@types/draft-js": "^0.11.9",
+        "@types/jsonwebtoken": "^8.5.8",
+        "@types/node": "^17.0.27",
+        "@types/react": "^18.0.7",
+        "@types/react-dom": "^18.0.2",
+        "@types/react-draft-wysiwyg": "^1.13.4",
+        "@types/react-redux": "^7.1.24",
+        "axios": "^0.26.1",
+        "date-fns": "^2.28.0",
+        "date-fns-tz": "^1.3.4",
+        "dayjs": "^1.11.2",
+        "draft-js": "^0.11.7",
+        "file-loader": "^6.2.0",
+        "formiojs": "^4.14.6",
+        "jsonwebtoken": "^8.5.1",
+        "keycloak-js": "^10.0.2",
+        "mui-sx": "^1.0.0",
+        "node-sass": "^7.0.1",
+        "react": "^18.0.0",
+        "react-dom": "^18.0.0",
+        "react-draft-wysiwyg": "^1.14.7",
+        "react-dropzone": "^14.2.1",
+        "react-redux": "^7.2.8",
+        "react-router-dom": "^6.3.0",
+        "react-scripts": "5.0.1",
+        "react-svg": "^15.0.1",
+        "react-test-renderer": "^18.2.0",
+        "redux": "^4.1.2",
+        "typescript": "^4.6.3",
+        "web-vitals": "^2.1.4"
+    },
+    "scripts": {
+        "start": "react-app-rewired start",
+        "build": "react-app-rewired build",
+        "test": "jest",
+        "eject": "react-scripts eject",
+        "lint": "eslint '*/**/*.{ts,tsx}'"
+    },
+    "eslintConfig": {
+        "extends": [
+            "react-app",
+            "react-app/jest"
+        ]
+    },
+    "browserslist": {
+        "production": [
+            ">0.2%",
+            "not dead",
+            "not op_mini all"
+        ],
+        "development": [
+            "last 1 chrome version",
+            "last 1 firefox version",
+            "last 1 safari version"
+        ]
+    },
+    "devDependencies": {
+        "@testing-library/jest-dom": "^5.16.4",
+        "@types/jest": "^27.5.1",
+        "@typescript-eslint/eslint-plugin": "^5.24.0",
+        "@typescript-eslint/parser": "^5.24.0",
+        "assert": "^2.0.0",
+        "buffer": "^6.0.3",
+        "crypto-browserify": "^3.12.0",
+        "eslint": "^8.15.0",
+        "eslint-config-prettier": "^8.5.0",
+        "eslint-plugin-prettier": "^4.0.0",
+        "https-browserify": "^1.0.0",
+        "jest": "^28.1.0",
+        "jest-environment-jsdom": "^28.1.0",
+        "jest-extended": "^2.0.0",
+        "os-browserify": "^0.3.0",
+        "prettier": "^2.6.2",
+        "process": "^0.11.10",
+        "react-app-rewired": "^2.2.1",
+        "stream-browserify": "^3.0.0",
+        "stream-http": "^3.2.0",
+        "ts-jest": "^28.0.3",
+        "ts-node": "^10.8.0",
+        "url": "^0.11.0",
+        "@emotion/react": "^11.9.0",
```

Comment:
I'm assuming you need all these dependencies in dev bc that's the configuration the tests run?

Reply:
yes